google/auth/external_account.py

```diff
@@ -52,6 +52,43 @@
 _CLOUD_RESOURCE_MANAGER = "https://cloudresourcemanager.googleapis.com/v1/projects/"
 
 
+class SupplierContext(object):
+    """A context object that contains information about the requested third party credential that is passed
+    to AWS security credential and subject token suppliers."""
+
+    def __init__(self, subject_token_type, audience):
```

Comment:
Audience could default to None if it is optional

Reply:
Whoops, forgot to fix the comments here from the original code I took from a different part of the repo. Audience is not optional.